docs/docs/spark-getting-started.md

```diff
@@ -188,6 +188,12 @@ This type conversion table describes how Iceberg types are converted to the Spar
 | struct                     | struct                  |               |
 | list                       | array                   |               |
 | map                        | map                     |               |
+| nanosecond timestamp       |                         | Not supported |
+| nanosecond timestamp with timezone |                 | Not supported |
+| unknown                    |                         | Not supported |
+| variant                    |                         | Spark 4.0+    | 
```

Comment:
shouldn't the spark type be `variant` (instead of empty)? The note column already mentioned 4.0+